pkgs/by-name/in/install-grub-ng/src/config.rs

```diff
@@ -0,0 +1,292 @@
+use std::{borrow::Cow, collections::HashMap, iter::Iterator, path::Path};
+
+use eyre::{Context, Result, bail, eyre};
+use roxmltree::{Document, Node};
+
+use crate::grub::FsIdentifier;
+
+config! {
+  'a;
+  grub: Option<&'a Path> => grub,
+  grub_target: Option<&'a Path> => grubTarget,
```

Comment:
Why not use serde?  
We could also use something else than XML when using NG.

Reply:
Serde doesn't really have a standard API that parses *normal* XML well, let alone the weird format that `builtins.toXML` generates. I think we should really just switch to a saner format like JSON